website/integrations/services/gitea/index.md

```diff
@@ -62,6 +62,45 @@ Change the following fields
 - Icon URL: https://raw.githubusercontent.com/goauthentik/authentik/master/web/icons/icon.png
 - OpenID Connect Auto Discovery URL: https://authentik.company/application/o/gitea-slug/.well-known/openid-configuration
 
-![](./gitea1.png)
+
+(./gitea1.png)
 
 `Add Authentication Source` and you should be done. Your Gitea login page should now have a `Sign in With` followed by the authentik logo which you can click on to sign-in to Gitea with Authentik creds.
+
+
+
+
+### Step 4 (Optional)
```

Comment:
I wouldn't add this as 4th step but rather as part of the gitea setup under step 3. Imo it's not optional (since a stock install won't work without this), so I'd say just add the `In your Gitea instance, navigate to your app.ini and make the following changes` block.